src/data/compat/Proton.vala

```diff
@@ -88,7 +88,14 @@ namespace GameHub.Data.Compat
 			else
 			{
 				File? proton_dir = null;
-				if(Steam.find_app_install_dir(appid, out proton_dir))
+                if(appid.has_prefix("custom")) {
+                    proton_dir = File.new_for_path(Environment.get_home_dir ()).get_child (".steam").get_child ("steam").get_child ("compatibilitytools.d").get_child (appname);
+                    name = appname;
+                    executable = proton_dir.get_child("proton");
+					installed = executable.query_exists();
+					wine_binary = proton_dir.get_child("dist").query_exists() ? proton_dir.get_child("dist/bin/wine") : proton_dir.get_child("files/bin/wine");
```

Comment:
This check is now necessary for the standard proton too (it is required for Proton Experimental) here https://github.com/tkashkin/GameHub/blob/9327885393f022fd1cccb219a19c0f87ae5e0f5a/src/data/compat/Proton.vala#L98

Reply:
I do not know how to merge this into my PR, do I just mark as resolved?